realm/realm-library/src/main/java/io/realm/DynamicRealmObject.java

```diff
@@ -755,6 +755,10 @@ public void setObject(String fieldName, @Nullable DynamicRealmObject value) {
 
     /**
      * Sets the reference to a {@link RealmList} on the given field.
+     * <p>
+     * This will copy all the elements in the list into Realm, but any further changes to the list
+     * will not be reflected in the Realm. Use {@link #getList(String)} in order to a reference to
```

Comment:
```suggestion
     * will not be reflected in the Realm. Use {@link #getList(String)} in order to get a reference to
```
Was that what you meant?